EIPS/eip-4361.md

```diff
@@ -22,20 +22,21 @@ Already, many services support workflows to authenticate Ethereum accounts using
 ## Specification
 Sign-In with Ethereum works as follows:
 
-1. The wallet presents the user with a structured plaintext message or equivalent interface for signing with the [EIP-191](./eip-191.md) signature scheme (string prefixed with `\x19Ethereum Signed Message:\n<length of message>`). The `message` MUST incorporate an Ethereum `address`,  `domain` requesting the signing, `version` of the message, a chain identifier `chain-id`, `uri` for scoping, `nonce` acceptable to the server, and `issued-at` timestamp.
-2. The signature is then presented to the server, which checks the signature's validity and message content.
-3. Additional fields, including `expiration-time`, `not-before`, `request-id`, `chain-id`, and `resources` may be incorporated as part of authentication for the session.
-4. The server may further fetch data associated with the Ethereum address, such as from the Ethereum blockchain (e.g., ENS, account balances, ERC-20/ERC-721/ERC-1155 asset ownership), or other data sources that may or may not be permissioned.
+1. The wallet presents the user with a structured plaintext message or equivalent interface for signing with the [EIP-191](./eip-191.md) signed data format. Before signing, the `message` is prefixed with `\x19Ethereum Signed Message:\n<length of message>` as defined in [EIP-191](./eip-191.md).
+The `message` MUST incorporate an Ethereum `address`,  `domain` requesting the signing, `version` of the message, a chain identifier `chain-id`, `uri` for scoping, `nonce` acceptable to the relying party, and `issued-at` timestamp.
+2. The signature is then presented to the relying party, which checks the signature's validity and message content.
+3. Additional fields, including `expiration-time`, `not-before`, `request-id`, `statement`, and `resources` MAY be incorporated as part of the sign-in process.
+4. The relying party MAY further fetch data associated with the Ethereum address, such as from the Ethereum blockchain (e.g., ENS, account balances, ERC-20/ERC-721/ERC-1155 asset ownership), or other data sources that might or might not be permissioned.
```

Comment:
The first time an EIP is referenced, it must be linked.

Also, prefer `EIP-*` over `ERC-*`